keyboards/ergotravel/keymaps/jpconstantineau/rules.mk

```diff
@@ -0,0 +1,6 @@
+RGBLIGHT_ENABLE = yes
+BACKLIGHT_ENABLE = yes
+
+ifndef QUANTUM_DIR
```

Comment:
This block is not needed.  It's a holdover from when we were still using makefiles here.

Reply:
I removed the *LIGHT lines from all rules files that were in the keymap folders.